factory/crypto/cryptoComponents.go

```diff
@@ -271,6 +271,11 @@ func (ccf *cryptoComponentsFactory) readCryptoParams(keygen crypto.KeyGenerator)
 		return nil, err
 	}
 
+	cp.privateKeyBytes, err = cp.privateKey.ToByteArray()
```

Comment:
these aren't used. Instead of properly setting the private key bytes, why not remove the field & functions in the corresponding interface?

Reply:
I was thinking about this too.. removed